jnosql-metamodel-extension/README.adoc

```diff
@@ -0,0 +1,4 @@
+= Metamodel Layer
+
+This extension contains Metamodel class definitions, used to describe a type annotated with JNoSql annotations such as Entity, MappedSupertype or Embeddable.
+These classes are not meant to be used directly, metamodel-processor will do that for you.
```

Comment:
Please, use the readme to explore more the idea about this extension.
What does the metamodel-extension do?
How can I use it?
Code sample.

Reply:
Should be ok now !